py/README.md

```diff
@@ -0,0 +1,87 @@
+# Deephaven Python Source
+
+This folder contains Deephaven Community Core's Python source code, tests, and more. Source code can be found in `server/deephaven` and unit tests can be found in `server/tests`. Other content includes `jpy`, the Python client, wheels, and more.
+
+## Requirements
+
+Development and contribution requires _at least_ Python 3.8 or later. We recommend always staying up-to-date on the latest Python version.
```

Comment:
I'm not sure staying up-to-date with the latest Python version is necessarily true for development since we have to ensure new releases still work, but I'm not the Python expert for DHC